<!DOCTYPE HTML>
<!--
	Spatial by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>

<head>
	<title>Sepideh Maleki | AI Research Postdoc fellow @ Genentech</title>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<noscript>
		<link rel="stylesheet" href="css/style.css" />
		<link rel="stylesheet" href="css/skel.css" />
		<link rel="stylesheet" href="css/style-xlarge.css" />
	</noscript>
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Raleway:400,700">
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
	<link rel="stylesheet" href="css/font-awesome.min.css">
	<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
	<script src="js/jquery.min.js"></script>
	<script src="js/skel.min.js"></script>
	<script src="js/skel-layers.min.js"></script>
	<script src="js/init.js"></script>
	<script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>
</head>

<body class="landing">

	<!-- Header -->
	<header id="header">
		<ul class="icons">
			<li>
				<a href="https://scholar.google.com/citations?user=JSwHqtYAAAAJ&hl=en" class="icon fa-graduation-cap"></a>
			</li>
				<li>
					<a href="https://github.com/sepidism" class="icon fa-github"></a>
				</li>
			<li>
				<a href="https://www.linkedin.com/in/sepideh-maleki-08507460" class="icon fa-linkedin"></a>
			</li>
		</ul>
		<nav id="nav">
			<ul>
				<li><a href="/" class="active">Home</a></li>
				<li><a href="/#two">Publications</a></li>
				<li><a href="/#footer">Contact</a></li>
			</ul>
		</nav>
	</header>

	<!-- One -->
	<section id="one" class="wrapper style1">
		<div class="container 75%">
			<div class="row 200%">
				<div class="6u 12u$(medium)">
					<header class="major">
						<h2>Sepideh Maleki</h2>
						<p>
							<div class="image rounded" style="margin-bottom: 1.5em;"><img src="sm.jpeg" width="200" alt="" /></div>
						</p>
						<p style="text-transform:none">AI Postdoc Fellow</p>
						<p style="text-transform:none">Genentech</p>
					</header>
				</div>
				<div class="6u$ 12u$(medium)">
          <p align="justify">I am currently a postdoctoral fellow at Genentech. My research is conducted within the department of BRAID, which specializes in applying machine learning to research biology. I am part of the "Foundational ML Research" team, where I have led and contributed to multiple research projects. 
		  My research focuses on the convergence of foundation models, graph theory, and machine learning, with a special emphasis on advancing biological and drug discovery applications. 
		  I am particularly engaged in developing machine learning methods that efficiently leverage foundation models—such as large language models and single-cell models—to tackle complex challenges in molecular and genetic perturbation.



Before joining Genentech, I was a PhD student in the Department of Computer Science at the University of Texas at Austin under supervision of <a href="https://www.cs.utexas.edu/~pingali/" target="_blank">Prof. Keshav Pingali</a>. 
				</div>
			</div>
		</div>

		<div class="container 75%">
			<h3 style="text-align: center">Highlights</h3>
			<ul>
	<li>03/2025: Efficient Fine-Tuning of Single-Cell Foundation Models Enables Zero-Shot Molecular Perturbation Prediction was accpeted as a spotlight at ICLR'25 (MLGENX)<a href="https://arxiv.org/abs/2412.13478/">Efficient Fine-Tuning of Single-Cell Foundation Models Enables Zero-Shot Molecular Perturbation Prediction</a> .</li>

	<li>01/2025: Our recent paper on Molecular perturbation is on ArXiv! <a href="https://arxiv.org/abs/2412.13478/">Efficient Fine-Tuning of Single-Cell Foundation Models Enables Zero-Shot Molecular Perturbation Prediction</a> .</li>

	<li>09/2024: Our recent paper on Score-based Explainability for Graph Representations is accepted at <a href="https://jmlr.org/tmlr/">Transactions on Machine Learning Research (TMLR)</a> .</li>
	<li>08/2024: Our recent paper on <a href="https://openreview.net/pdf?id=DnG75_KyHjX">Leveraging Single-Cell Foundation Models for Molecular Perturbation Prediction</a> is accepted at MLCB 2024.</li>
        <li>03/2024: Our paper on <a href="https://www.nature.com/articles/s41598-024-55934-5">Community detection in hypergraphs via mutual information maximization</a> is accepted at Nature Scientific Reports. 
        <li>02/2024: Our proposal for the 1st Machine Learning for Genomics Exploration Workshop (MLGenX) <a href="https://mlgenx.github.io/"> was accepted at ICLR24.</a></li>
        
			</ul>
		</div>
	</section>

	<!-- Two -->
<!-- 	<section id="two" class="wrapper style2">
		<div class="container 75%">
			<h2 style="text-align: center">Selected Publications</h2>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/bayrel.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Bayesian Relational Learning</h4>
						<p>a novel Bayesian representation learning method that infers the relational interactions across multi-omics data types.</p>
					</header>
					<p><span style="text-decoration: underline;">E. Hajiramezanali*</span>, A. Hasanzadeh*, N. Duffield, K. Narayanan, and X. Qian, <span style="font-weight: bold">BayReL: Bayesian Relational Learning for Multi-omics Data Integration</span>, NeurIPS 2020, [<a href='https://proceedings.neurips.cc/paper/2020/file/df5511886da327a5e2877c3cd733d9d7-Paper.pdf'>NeurIPS</a>, <a href='https://arxiv.org/abs/2010.05895'>arXiv:2010.05895</a>,<a href="http://ehsanhajiramezanali.github.io/misc/bayrelSlides.pdf">Slides</a>, <a href="http://ehsanhajiramezanali.github.io/misc/bayrelPoster.pdf">Poster</a>, <a href="https://github.com/ehsanhajiramezanali/BayReL">Code</a>], *equal contribution.</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/gcn2.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Graph DropConnect</h4>
						<p>A unified framework for adaptive connection sampling in GNNS that not only alleviates over-smoothing and over-fitting tendencies of deep GNNs, but also enables learning with uncertainty.</p>
					</header>
          <p>A. Hasanzadeh*, <span style="text-decoration: underline;">E. Hajiramezanali*</span>, M. Zhou, N. Duffield, K. Narayanan, and X. Qian, <span style="font-weight: bold">Bayesian Graph Neural Networks with Adaptive Connection Sampling</span>, ICML 2020, [<a href='https://proceedings.icml.cc/static/paper_files/icml/2020/6133-Paper.pdf'>ICML</a>, <a href='https://arxiv.org/abs/2006.04064'>arXiv:2006.04064</a>,<a href="http://ehsanhajiramezanali.github.io/misc/gdcSlides.pdf">Slides</a>, <a href="https://github.com/ehsanhajiramezanali/GDC">Code</a>], *equal contribution.</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/vgrnn.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Variational Graph Recurrent Neural Networks</h4>
						<p>Node embedding method for dynamic graphs that maps each node to a random vector in the latent space.</p>
					</header>
          <p><span style="text-decoration: underline;">E. Hajiramezanali*</span>, A. Hasanzadeh*, K. Narayanan, N. Duffield, M. Zhou, and X. Qian, <span style="font-weight: bold">Variational Graph Recurrent Neural Networks</span>, NeurIPS 2019, [<a href='http://papers.nips.cc/paper/9254-variational-graph-recurrent-neural-networks'>NeurIPS</a>, <a href='https://arxiv.org/abs/1908.09710'>arXiv:1908.09710</a>,<a href="http://ehsanhajiramezanali.github.io/misc/vgrnnPoster.pdf">Poster</a>, <a href="https://github.com/ehsanhajiramezanali/VGRNN">Code</a>], *equal contribution.</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/sobc.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Scalable Optimal Bayesian Classification</h4>
						<p>Optimal classification of single-cell trajectories accounting for potential model uncertainty.</p>
					</header>
					<p><span style="text-decoration: underline;">E. Hajiramezanali</span>, M. Imani, U. Braga-Neto, X. Q, and E. Dougherty, <span style="font-weight: bold">Scalable Optimal Bayesian Classification of Single-Cell Trajectories under Regulatory Model Uncertainty</span>, BMC Genomics, [<a href='https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5720-3'>BMC Genomics</a>, <a href='http://ehsanhajiramezanali.github.io/misc/sobc.pdf'>PDF</a>].</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/bmdl.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Bayesian Multi-Domain Learning</h4>
						<p>Multi-domain negative binomial factorization model for over-dispersed count data.</p>
					</header>
					<p><span style="text-decoration: underline;">E. Hajiramezanali</span>, S. Z. Dadaneh, A. Karbalayghareh, M. Zhou, and X. Qian, <span style="font-weight: bold">Bayesian Multi-Domain Learning for Cancer Subtype Discovery from Next-Generation Sequencing Count Data</span>, NeurIPS 2018, [<a href='http://papers.nips.cc/paper/8125-bayesian-multi-domain-learning-for-cancer-subtype-discovery-from-next-generation-sequencing-count-data'>NeurIPS</a>, <a href='http://ehsanhajiramezanali.github.io/misc/Ehsan_BMDL_NIPS2018.pdf'>PDF</a>].</p>
				</div>
			</div>
			<hr />
			<p style="text-align:center;">For a full list, have a look at my <a href="https://scholar.google.com/citations?user=20I_DMoAAAAJ&hl=en">Google Scholar</a> page.</p>
		</div>
	</section>
 -->

	<!-- Three -->
	<section id="three" class="wrapper style3 special">
		<div class="container 50%">

			<h2>Curriculum vitae</h2>
			<ul class="actions">
				<li><a href="CV_SepidehMaleki.pdf" class="button alt big">Download CV</a></li>
			</ul>
		</div>
	</section>


	<!-- Footer -->
	<footer id="footer">
		<div class="container">
			<h2>Get in touch</h2>
			<p><span style="font-weight: bold">Sepid [DOT] Maleki [AT] GMAIL [DOT] COM</span><br /> Foundational ML Research, Genentech<br />Research and Early Development (gRED), AI/ML</p>
			<p></p>
			<ul class="icons">
			<li>
				<a href="https://scholar.google.com/citations?user=20I_DMoAAAAJ&hl=en" class="icon fa-graduation-cap"></a>
			</li>
				<li>
					<a href="https://github.com/sepidism" class="icon fa-github"></a>
				</li>
			<li>
				<a href="https://www.linkedin.com/in/ehsan-hajiramezanali-978a3b52/" class="icon fa-linkedin"></a>
			</li>
			</ul>
			<ul class="copyright">
				<li>&copy; 2024 Sepideh Maleki</li>
				<li>Design: <a href="http://templated.co">TEMPLATED</a></li>
			</ul>
		</div>
	</footer>

</body>

</html>
